common/config/startup.nsh

```diff
@@ -66,6 +66,10 @@ endfor
 :DoneDebug
 
 for %j in 0 1 2 3 4 5 6 7 8 9 A B C D E F then
+    if exist FS%j:\EFI\BOOT\bsa\sr_bsa.flag then
+        FS%j:\EFI\BOOT\bsa\sbsa.nsh
+        goto Donebsa
+    endif
     if exist FS%j:\EFI\BOOT\bsa\bsa.nsh then
```

Comment:
In the order of running, can we run bsa first and then sbsa?

Reply:
sbsa.nsh runs BSA binary first and then SBSA